napari/_app_model/constants/_commands.py

```diff
@@ -64,6 +68,9 @@ class _i(NamedTuple):
 
 # fmt: off
 _COMMAND_INFO = {
+    CommandId.TOGGLE_CONSOLE: _i(trans._('console')),
```

Comment:
Would be great if we could standardize caps or no caps for stuff like this. I know it's nitpicky but consistency is good. Let's go with capitalizing each word? I feel like that's very common